app/controllers/oaisys/pmh_controller.rb

```diff
@@ -278,4 +279,48 @@ def prep_identifiers(parameters)
     parameters
   end
 
+  # Returning nil gives a bad argument error.
+  def handle_from_and_until_dates(model, from_date, until_date)
+    if from_date.present?
+      from_date_format = get_date_format(from_date)
+
+      case from_date_format
+      when 1
```

Comment:
Just leaving a note here for January about what we discussed re: replacing these ints with meaningful symbols to make this a bit clearer to the reader